### ISSUE TITLE
OAuth with Github does not work

### ISSUE THREAD

**Issue by schmidsi** [kactus-io/kactus#5]
### Description

Sign-in using the browser does not work

### Version

**Kactus version:** v0.1.0-alpha6
**OS version:** macOS Sierra Version 10.12.5 (16F73)

### Steps to Reproduce

1. Download Kaktus
1. Open it
1. Click on "Sign into GitHub.com"
1. Click on "Sign in using your browser" --> The browser opens: `https://github.com/login/oauth/authorize?client_id=`
1. Click on "Authorize Kactus"
1. Browser goes to `https://kactus.io/auth/github?code=f...`

![screen shot 2017-06-30 at 11 02 34](https://user-images.githubusercontent.com/103280/27729117-ccb6776c-5d84-11e7-8583-8957adfcfda0.png)

**Expected behaviour:**
- Site is properly secured
- The url somehow redirects so that the Kactus app receives the OAuth-Token

**Actual behaviour:**
- Chrome brings a security warning
- Site shows a 404 error

**Reproduces how often:**
100%

### Logs

There are no logs. Here is an excerpt from: `~/Library/Application Support/Kactus`

```
GPUCache
IndexedDB
Local Storage
Preferences
QuotaManager
QuotaManager-journal
SS -> /var/folders/mc/.../T/.io.kactus.KactusClient..../SS
SingletonCookie -> 4617403899741648914
SingletonLock -> simonschmids-macbook.local-...
databases
window-state.json
```

**Comment by schmidsi** [comment]
Login with Username/Password + 2FA worked.

**Comment by mathieudutour** [comment]
oops, should be `https://api.kactus.io`. Will fix asap

**Comment by ahuk** [comment]
Same here, but the URL shown in Safari after granting access starts with
`x-kactus-auth://oauth?code=`

**Comment by mathieudutour** [comment]
yep, just need to cut off a new release (I fixed the backend but didn't release the frontend yet), will do it today hopefully

**Comment by mathieudutour** [comment]
fixed in 0.1.0
